src/pages/about-us/constants.ts

```diff
@@ -0,0 +1,126 @@
+import type { DeveloperInfo, Role, RoleDescription } from '../../app/types';
+
+import { HEADER2 } from '../../shared/styles';
+import { CARD } from '../catalog/constants';
+
+export const TEAM_SLOGAN = '🚀 Rocket Team: We came. We coded. We conquered. 🌟';
+
+export const RSS_LINK = 'https://rs.school/courses';
+
+export const LABEL_RSS_LINK = 'Our alma mater:';
+
+const ROLES: Record<Role, RoleDescription> = {
+  testing: {
+    label: '🧪 Testing',
+    color: 'bg-red-200',
+  },
+  team_lead: {
+    label: '👑 Team lead',
+    color: 'bg-orange-200',
+  },
+  chores: {
+    label: '🧹 Chores',
+    color: 'bg-indigo-200',
+  },
+  api: {
+    label: '🛠️ API',
+    color: 'bg-lime-200',
+  },
+  sdc: {
+    label: '🚀 CT-SDK',
+    color: 'bg-sky-200',
+  },
+  ui: {
+    label: '🎨 UI & Content',
+    color: 'bg-pink-200',
+  },
+  routing: {
+    label: '🚦 Routing',
+    color: 'bg-fuchsia-200',
+  },
+  architecture: {
+    label: '🏛️ Architecture',
+    color: 'bg-rose-200',
+  },
+};
+
+export const DEVELOPERS: DeveloperInfo[] = [
+  {
+    photoURL: '/jpg/maria.jpg',
+    name: 'Maria',
+    roles: [ROLES.team_lead, ROLES.api, ROLES.architecture, ROLES.ui],
+    description:
+      'Zdravstvuyte!\n\n I’m @ihavenomouthbutimustscreamm, 25 years old and I’m from Russia.\nGerman tutor and a passionate linguist. My first dive into coding was customizing a Hannibal Tumblr fanpage with HTML and CSS back in my teenage years. Years later, while writing my graduation thesis, I stumbled upon RSSchool courses and joined just for fun. Now I’m a code junkie - can’t go a day without tweaking, building, or breaking something in my apps.\n\nContribution:\n - Login page 🔑\n - Catalog page 🖼️\n - Cart page 🛒',
+    github: 'https://github.com/koonukaame',
+  },
+  {
+    photoURL: '/jpg/tanya.jpg',
+    name: 'Taniana Grosul',
```

Comment:
typo. Unless I don't know something about you :D
```suggestion
    name: 'Tatiana Grosul',
```

Reply:
done